developer-docs/bolt-api-servers.md

```diff
@@ -152,6 +152,7 @@ object](https://github.com/puppetlabs/puppetserver/blob/master/documentation/pup
 from [puppetserver](https://github.com/puppetlabs/puppetserver), with an
 additional `file_content` key.
 
+See the [schema](../lib/bolt_ext/schemas/task.json)
```

Comment:
ugh. I copy pasted these from above. They all point to the old location... Will wait for any other comments before fixing up to save some CI load.